core/pta/tests/fs_htree.c

```diff
@@ -433,7 +433,7 @@ static struct test_aux *aux_alloc(size_t num_blocks)
 static TEE_Result test_write_read(size_t num_blocks)
 {
 	struct test_aux *aux = aux_alloc(num_blocks);
-	TEE_Result res;
+	TEE_Result res = TEE_SUCCESS;
```

Comment:
Note that currently this function is called with a non-zero argument, so we should not expect issue with current implementation. Nevertheless we should always have an init value for such local variables.

By the way, could you also add an init value to the 3 variables right below? Actually, the whole source file should be updated.

Reply:
> By the way, could you also add an init value to the 3 variables right below? Actually, the whole source file should be updated.

I meant such change (updating the whole source file for local vars initialization) may not in the scope of your P-R. Tell us if so, we will create another P-R to address that.

Reply:
> By the way, could you also add an init value to the 3 variables right below?

I don't think they are as critical as `res`, since `n`, `m`, `o` are just loop counters. But ok.

> updating the whole source file for local vars initialization

I'll add a commit to address that.